packages/app-bridge/src/types/Terrific.ts

```diff
@@ -21,6 +22,10 @@ export type TemplateChooserResult = TemplateApiLegacy;
 
 export type AssetChooserAssetChosenCallback = (selectedAsset: { screenData: AssetChooserResult[] }) => void;
 export type TemplateChooserTemplateChosenCallback = (selectedTemplate: { template: TemplateChooserResult }) => void;
+export type CreateNewPublicationOptions = {
+    template: Template;
+    projectId: number;
```

Comment:
Please have a look to the web-app comment

Reply:
Cleaned up the `projectId` prop. Thanks for hint!